File: PHP/RA2_2.1_6_DanielTorres.php
<?php
// Programa fet per Daniel Torres
// Data: 16/10/2023
// Versió: 2.1
// Et diu si són vocals o consonants

// Demana una lletra a l'usuari
echo "Introdueix una lletra: ";
$lletra = trim(fgets(STDIN));

// Comprova si la lletra és una vocal
if (in_array($lletra, ['a', 'e', 'i', 'o', 'u'])) {
    echo "La lletra és una vocal\n"; // Missatge si és vocal
} else {
    echo "La lletra és una consonant\n"; // Missatge si és consonant
}
?>

File: R2/2.1 ControlFlux 1/RA2_2.1_6_DanielTorres.py
#Programa fet per Daniel Torres
#Data: 02/10/2025
#Versio: 2.1
#Te diu si son vocals o consonants
lletra = input("Introdueix una lletra: ") #Variable que demana una lletra
if(lletra in ['a','e','i','o','u']): #Condicio que comprova si la lletra es una vocal
              print("La lletra es una vocal") #Missatge que surt si es una vocal
else:
              print("La lletra es una consonant") #Missatge que surt si es una consonant